content/CPPLINT.cfg

```diff
@@ -0,0 +1,4 @@
+filter=-whitespace/tab
+filter=-whitespace/line-length
+filter=-whitespace/braces
+filter=-legal/copyright
```

Comment:
What about `filter=-build/include_subdir`? It's so that you don't have to include the full path to `utils.h` or `log.h`. I think it makes sense.

Reply:
I could have sworn I mentioned that. Yes, I'm thinking that including `common/util.h` and `common/log.h` does make sense. Because it provides some sort of "namespacing" of header files. But I won't advocate that too much. if you feel it doesn't make sense, we'll add the `filter=-build/include_subdir` option as well.

Reply:
Ok, let it be.

Reply:
@teodutu, we'll update the includes in Makefiles, as discussed. We'll keep this filter.